### PR TITLE
Take the rook images from registry.suse.com

### PR DESCRIPTION
**Pull request by jsuchome**
Remove the temporary hack that pointed to registry.suse.de

### CODE REVIEW

**Comment by jsuchome** [comment]
Thanks @Itxaka , updated

**Comment by Itxaka** [comment]
Need to run this over the ses_rook path to check if it indeed works

**Comment by jsuchome** [comment]
Yep, let's wait for https://github.com/SUSE-Cloud/socok8s/pull/731

**Comment by Itxaka** [comment]
> Yep, let's wait for #731

I actually retrigger the job with the ses_rook path :)

**Comment by jsuchome** [comment]
I do not see the CI results, but manual testing reveals:

```Failed to pull image "registry.suse.com/ses/6/ceph/ceph:14.2.2.354": rpc error: code = Unknown desc = Error reading manifest 14.2.2.354 in registry.suse.com/ses/6/ceph/ceph: manifest unknown: manifest unknown```

**Comment by jsuchome** [comment]
Updated SES repo, let's see if it changes things...

File: playbooks/roles/ses-rook/defaults/main.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ses_rook_repos:
-  ses_rook_sle15_sp1: "http://download.suse.de/ibs/Devel:/Storage:/6.0/SLE_15_SP1/"
+  ses_rook_sle15_sp1: "http://download.suse.de/ibs/SUSE:/SLE-15-SP1:/Update:/Products:/SES6:/Update/standard/"
```
Comment (AbelNavarro) [inline playbooks/roles/ses-rook/defaults/main.yml]: we probably want to use the provo mirror (when it works again): https://github.com/SUSE-Cloud/socok8s/pull/744/files#diff-0d7e3c79df5eb0ccb93627cfaa14b956L23

Comment (jsuchome) [reply playbooks/roles/ses-rook/defaults/main.yml]: Updated to use the variable, please check again

**Comment by ilausuch** [comment]
Tested (manually). This works!

**Pull request merged**